### ISSUE TITLE
strange bug/feature: when exporting video, it puts whatever sound you happen to be listening to in another application on the export

### ISSUE THREAD

**Issue by df-0** [nvrmind/IssueTracker#60]
in this case, unexpected hiphop when I play back the exported video. Let's say it's...a feature :-)

**Comment by daseyb** [comment]
Yeah, we’re using a third party plugin to do the mp4 video export and it
turns out they just record system sound.
We’ll replace that at some point
